Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -303,6 +332,41 @@ def runSelected(opt):
                       default=None,
                       action='store')
 
+    parser.add_option('--RequiresGPU',
+                      help='if GPU is reuired or not: forbidden (default, CPU-only), optional, required',
+                      dest='RequiresGPU',
+                      default=0)
+
+    parser.add_option('--GPUMemory',
+                      help='to specify GPU memory. Default = 8000 MB (for RequiresGPU = required).',
+                      dest='GPUMemory',
+                      default='8000')
+
+    parser.add_option('--CUDACapabilities',
+                      help='to specify CUDA capabilities. Default = 7.5 (for RequiresGPU = required).',
+                      dest='CUDACapabilities',
+                      default='7.5')
+
+    parser.add_option('--CUDARuntime',
+                      help='to specify CUDA runtime. Default = 11.2 (for RequiresGPU= required).',
+                      dest='CUDARuntime',
+                      default='11.2')
```

Comment:
Should this also default to whatever the release uses?

Reply:
Yeah, that can be done. I just put the default one here to make sure that it will not be an empty field when GPU is required.

Reply:
Something like
```bash
scram tool info cuda | sed -n -e's/^Version *: *\([[:digit:]]\+\.[[:digit:]]\+\)\.[[:digit:]]\+/\1/p'
```

Though maybe we should add an environment variable in `cuda.spec` for this ?